build_deploy.sh

```diff
@@ -0,0 +1,33 @@
+set -exv
+
+IMAGE="quay.io/cloudservices/content-sources-backend"
+IMAGE_TAG=$(git rev-parse --short=7 HEAD)
```

Comment:
```suggestion
IMAGE_TAG="$(git rev-parse --short=7 HEAD)"
```
I would use quotes `"` to enclose variable value.